chart/validator/values.yaml

```diff
@@ -71,7 +71,11 @@ proxy:
   secretName: proxy-cert
   # If false (using an existing secret), the key whose value is the CA certificate must end with '.crt'.
   createSecret: false
-  caCert: ""  # Raw CA certificate, required if createSecret is true
+  # Raw CA certificate, required if createSecret is true
+  caCert: |
+    -----BEGIN CERTIFICATE-----
+    <your certificate content here>
+    -----END CERTIFICATE-----
```

Comment:
imo adding a cert like this has a little bit better ux than the previous setup. open for discussions though